capstone/src/main/java/com/google/univiz/api/VisResource.java

```diff
@@ -1,7 +1,20 @@
 package com.google.univiz.api;
 
+import com.google.univiz.CollegeData;
+import com.google.univiz.api.dataviz.Deadline;
+import com.google.univiz.api.dataviz.Timeline;
+import com.google.univiz.api.dataviz.TuitionStats;
+import java.util.List;
+
 /**
  * The VisResource interface is responsible for all data management/processing specific for the vis
  * feature of the UniViz webapp.
  */
-public interface VisResource {}
+public interface VisResource {
+
+  List<TuitionStats> getTuitionStats(List<CollegeData> colleges);
```

Comment:
Each fxn in the interface should have it's own javadoc

Reply:
Ah good catch, I forget to deal with JavaDoc.